packages/spindle-ui/src/Pagination/Pagination.tsx

```diff
@@ -81,13 +79,15 @@ export const Pagination = (props: Props) => {
         {displayItem.map((pageNumber, index) => {
           const isCurrent = current === pageNumber;
           const hasRelAttribute = current === pageNumber + 1;
+          const showPrevMenuHorizontal = index === 0 && showPrevHorizontal;
+          const showNextMenuHorizontal = index === 5 - 1 && showNextHorizontal;
```

Comment:
5というマジックナンバーから抜け出せてないので、やるならこうでしょうか？(表示要素の最後)(要素が最大件数に満たない場合はshowXXXが既にfalseだと思うので)
```suggestion
          const showNextMenuHorizontal = index === displayItem.length - 1 && showNextHorizontal;
```

Reply:
たしかにぃぃ。修正しておきました！

Reply:
ありがとうございます！
あとはスタイル側の対応(?)を待ちます🙇